helix-core/src/comment.rs

```diff
@@ -38,18 +38,22 @@ fn find_line_comment(
 }
 
 #[must_use]
-pub fn toggle_line_comments(doc: &Rope, selection: &Selection) -> Transaction {
+pub fn toggle_line_comments(
+    doc: &Rope,
+    selection: &Selection,
+    token: Option<String>,
```

Comment:
Nit: Make this `Option<&str>`, then you can `token.unwrap_or("//")` without requiring that `to_owned`.